### PR TITLE
LandingPage added

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Screenshot (4)

### CODE REVIEW

**Review by MiheerTamkhane — APPROVED** [review]
Good and Clean landing page.

File: docpage-CSS/typography.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&display=swap');
+
+
+
+body{
+font-family: 'Bree Serif', serif;
+font-family: 'Montserrat', sans-serif;
+font-family: 'Roboto Slab', serif;
+font-family: 'Recursive', sans-serif;
+}
```
Comment (mimansha-swarup) [inline docpage-CSS/typography.css:11]: I am not getting why u gave all this font-family 
plus you can make variables for font-family as you did for colors that will be handy

**Review by mimansha-swarup — COMMENTED** [review]
everything else looks good

**Review by vijaytembugade — APPROVED** [review]

File: docpage-CSS/typography.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply docpage-CSS/typography.css:11]: Okay I will look at this.Thankyou for your valuable feedback.

**Pull request merged**